packages/is-shallow-equal/src/objects.js

```diff
@@ -1,6 +1,4 @@
-'use strict';
-
-var keys = Object.keys;
+const keys = Object.keys;
```

Comment:
`keys` appears to be used twice, stylistic but I'd prefer to just write `Object.keys` twice 🙂